theme/asset-editor.less

```
@@ -242,4 +242,18 @@
     i.fas {
         transform: rotate(90deg);
     }
+}
+
+.palette-transparent-button.common-button {
+    background: none transparent;
```

doesn't matter much but `background: none` is sufficient / clears out other all the other background-* sub properties for you so transparent shouldn't be needed~

```suggestion
    background: none;
```

---

thanks! I'll make that change in both spots.